From e3a52a48a749440f2fcb17dbd9caee7dbc636ee2 Mon Sep 17 00:00:00 2001
From: Vishal Sagar <vishal.sagar@xilinx.com>
Date: Fri, 23 Oct 2020 05:08:35 -0700
Subject: [PATCH 1722/1852] v4l: xilinx: m2m: Fix spelling of driver model

commit aaf416e254db079050259671880b0ee0c49e6d8d from
https://github.com/Xilinx/linux-xlnx.git

Fix spelling mistake in driver model.
s/Videoi/Video

Signed-off-by: Vishal Sagar <vishal.sagar@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/media/platform/xilinx/xilinx-m2m.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/media/platform/xilinx/xilinx-m2m.c b/drivers/media/platform/xilinx/xilinx-m2m.c
index 97f6db39f881..d9e16699abf6 100644
--- a/drivers/media/platform/xilinx/xilinx-m2m.c
+++ b/drivers/media/platform/xilinx/xilinx-m2m.c
@@ -1578,7 +1578,7 @@ static int xvip_composite_v4l2_init(struct xvip_m2m_dev *xdev)
 	int ret;
 
 	xdev->media_dev.dev = xdev->dev;
-	strlcpy(xdev->media_dev.model, "Xilinx Videoi M2M Composite Device",
+	strlcpy(xdev->media_dev.model, "Xilinx Video M2M Composite Device",
 		sizeof(xdev->media_dev.model));
 	xdev->media_dev.hw_revision = 0;
 
-- 
2.31.1

